kolibri/core/assets/src/views/ExamReport/__tests__/TriesOverview.spec.js

```diff
@@ -65,4 +78,111 @@ describe('TriesOverview', () => {
       expect(screen.getByText('notStartedLabel')).toBeInTheDocument();
     });
   });
+
+  describe("Test the 'Best Score' table row", () => {
+    test('renders the best score when there are past tries', () => {
+      renderComponent({
+        pastTries: [
+          {
+            id: '1',
+            correct: 8,
+            time_spent: 100,
+          },
+          {
+            id: '2',
+            correct: 9,
+            time_spent: 100,
+          },
+        ],
+        totalQuestions: 10,
+      });
+
+      expect(screen.getByText('Best score')).toBeInTheDocument();
+      expect(screen.getByText('90%')).toBeInTheDocument();
+
+      expect(screen.getByText('questionsCorrectLabel')).toBeInTheDocument();
+      expect(screen.getByText('questionsCorrectValue correct=9, total=10')).toBeInTheDocument();
+    });
+
+    test('renders the best score as 0 when there are no past tries', () => {
+      renderComponent();
```

Comment:
We can pass the `pastTries` explicitly as an empty array and the `totalQuestions` value.

Reply:
Got it! Thanks for the review!

Reply:
It seems that we forgot to update the code here :sweat_smile: I meant to add something like:

```javascript
renderComponent({ pastTries: [] });
```

Reply:
My bad! I totally missed this comment. Will update.

Reply:
Thank you!